package.json

```diff
@@ -41,6 +41,7 @@
     "@upsetjs/react": "^1.6.2",
     "bootstrap": "^4.6.0",
     "column-resizer": "^1.3.5",
+    "downshift": "^6.1.2",
```

Comment:
This package is for the UserSearchBox component to auto-suggest the accessors. https://github.com/downshift-js/downshift